modules/custom/d_p/js/d_p_ckeditor.js

```diff
@@ -0,0 +1,62 @@
+(function ($, Drupal) {
+  'use strict';
+  Drupal.behaviors.customCKEditorConfig = {
+    attach: function (context, settings) {
+
+      d_p_ckeditor(true);
+
+      function d_p_ckeditor(fade_in) {
+        fade_in = (fade_in == true) ? 500 : 0;
```

Comment:
Może byśmy tu zrobili pętlę - jeśli na stronie jest jakiś textarea z potencjalnym ckeditorem (może jest jakaś klasa albo coś w ten deseń), to wtedy próbuj co 500ms do skutku dodać do niego ten css. Obawiam się, że teraz CKEditor może nie zdążyć się zainicjalizować